make/autoconf/flags-cflags.m4

```diff
@@ -204,10 +204,15 @@ AC_DEFUN([FLAGS_SETUP_QUALITY_CHECKS],
       # This is most likely not really correct.
 
       # Add runtime stack smashing and undefined behavior checks.
-      CFLAGS_DEBUG_OPTIONS="-fstack-protector-all --param ssp-buffer-size=1"
-      CXXFLAGS_DEBUG_OPTIONS="-fstack-protector-all --param ssp-buffer-size=1"
+      # Not all versions of gcc support -fstack-protector-all
+      STACK_PROTECTOR_ALL_CFLAG="-fstack-protector-all"
+      FLAGS_COMPILER_CHECK_ARGUMENTS(ARGUMENT: [STACK_PROTECTOR_ALL_CFLAG -Werror],
```

Comment:
You don't need to add `-Werror` anymore, `$CFLAGS_WARNINGS_ARE_ERRORS` is now added automatically by `FLAGS_COMPILER_CHECK_ARGUMENTS` (as it should have been from the beginning).

Reply:
Fixed!